router.php

```diff
@@ -62,7 +62,7 @@
 		}
 
 		// Controller for the company page
-		$company = WebApp\Data::getCompanyInfo(str_replace('/', '', $url), $language->code);
+		$company = WebApp\Data::getCompanyInfo(str_replace('/', '', $url), strtoupper($language->code));
```

Comment:
Can you move the `strtoupper` function inside the `DataAccess\Company::selectPermalink` method instead?
Because the router is not supposed to knows this kind of detail about the DataAccess implementation.

Reply:
Updated this in latest commit